include/aws/auth/credentials.h

```diff
@@ -406,6 +406,23 @@ struct aws_credentials_provider_chain_default_options {
     struct aws_client_bootstrap *bootstrap;
 };
 
+/**
+ * For language binding to provide credentials from that language
+ */
+struct aws_credentials_provider_revert_options {
```

Comment:
"revert" as a descriptor doesn't seem like a good match.  "delegating" or "delegate" feel like it'd be pretty good.  "virtual" might be ok as well.   Others, like "proxy" are too overloaded with additional meanings.

Reply:
delegate it is!